index.js

```diff
@@ -14,7 +14,8 @@ class PackDir {
             dmg: /darwin/,
             dmgFormat: 'UDZO',
             isSilent: false,
-            isSync: true
+            isSync: true,
```

Comment:
Good, let's do it this way: main goal was to skip extra path inside ZIP files by default, so this flag is to be set to `true`.

Also, there's a check for `pathWithMask` on line 195 - plase, make in just `pathInfo.base` without extra check. Code will be more readable as well then.